docs/source/common/debugging.rst

```diff
@@ -97,8 +106,73 @@ argument of :class:`~pytorch_lightning.trainer.trainer.Trainer`)
 When using this flag, validation will be disabled. We will also replace the sampler
 in the training set to turn off shuffle for you.
 
+
+----------------
+
+
+************
+Optimization
+************
+
+Inspect Gradient Norms
+======================
+
+Logs (to a logger), the norm of each weight matrix.
```

Comment:
No, it's not the weight matrix at all. This is taking the norm of the gradients. and it is not a matrix in general.

Reply:
good catch!